### DISCUSSION TITLE
Can I use auth0 as auth provider ?

### DISCUSSION THREAD

**Discussion by lakshmajee** [vendurehq/vendure#1816] [Q&A]
Will I be able to user Auth0 jwt token validation into vendure? If yes how can we achieve that with vendure?

**Comment by michaelbromley** [comment]
Hi,
It should be possible. Take a look at these docs: https://www.vendure.io/docs/developer-guide/authentication/

**Reply by lakshmajee** [reply]
Thanks , will have a look at it

**Reply by fre2mansur** [reply]
Anyone could give me the step-by-step creation of a plugin for auth0 for storefront?, that would be helpful, instead of creating it separately as google, facebook etc?
